index.html

```diff
@@ -59,11 +59,11 @@ <h1 class="intro_2">
       </div>
 
       <ul class="social_media_icons">
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/twitter.png" alt="twitter_bird" /></a></li>
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/M_icon.jpg" alt="M_picture" /></a></li>
+        <li><a href="https://twitter.com/home"><img class="head_img" src="images/twitter.png" alt="twitter_bird" /></a></li>
+        <li><a href="https://analytics.google.com/analytics/web/#/provision"><img class="head_img" src="images/M_icon.jpg" alt="M_picture" /></a></li>
         <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/Linkedin.jpg" alt="linkedIn_icon" /></a></li>
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/github.jpg" alt="github_icon" /></a></li>
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/hand_icon.jpg" alt="hand" /></a></li>
+        <li><a href="https://github.com/BonkeGcobo/Portfolio"><img class="head_img" src="images/github.jpg" alt="github_icon" /></a></li>
+        <li><a href="https://www.w3.org/WAI/test-evaluate/preliminary/#contrast"><img class="head_img" src="images/hand_icon.jpg" alt="hand" /></a></li>
```

Comment:
- [ ] Unexpected movement by your content inside the red area (see the image below) appears when I hover your social media icons in the desktop layout. Kindly fix it.

![image](https://user-images.githubusercontent.com/60164239/131633856-29d79b0d-f946-4e1f-a018-24bdc30c5103.png)

Rule:

![image](https://user-images.githubusercontent.com/60164239/131634243-abb817c7-d7f0-4fd5-be20-9d71f2c2ca8f.png)